mne/io/nirx/nirx.py

```diff
@@ -28,6 +28,8 @@ def read_raw_nirx(fname, preload=False, verbose=None):
     ----------
     fname : str
         Path to the NIRX data folder or header file.
+    nosatflags : bool
+        Decide to use *.nosatflags_wlX instead of standard *.wlX files
```

Comment:
Can you expand this to describe what causes these files to be created (saturated values) and what the effect of the flag has (NaNs vs Saturated data). And mention that it is only relevant for NIRSport devices.

No need to be overly brief here, better to be longer and clearer.